sdk/resourcemanager/compute/armcompute/setup_test.go

```diff
@@ -0,0 +1,137 @@
+package armcompute_test
+
+import (
+	"context"
+	"fmt"
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore"
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/arm"
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/policy"
+	"github.com/Azure/azure-sdk-for-go/sdk/azidentity"
+	"github.com/Azure/azure-sdk-for-go/sdk/internal/recording"
+	"github.com/stretchr/testify/require"
+	"hash/fnv"
+	"net/http"
+	"os"
+	"testing"
+	"time"
+)
+
+var pathToPackage = "sdk/resourcemanager/compute/armcompute"
+
+func TestMain(m *testing.M) {
+	// Initialize
+
+	// Run
+	exitVal := m.Run()
+
+	// cleanup
+	os.Exit(exitVal)
+}
```

Comment:
You don't have to have this, you only need this if your adding recording sanitizers (which you will probably need to do)